Userland/Libraries/LibCore/SystemServerTakeover.cpp

```diff
@@ -18,12 +18,13 @@ static void parse_sockets_from_system_server()
 
     constexpr auto socket_takeover = "SOCKET_TAKEOVER";
     char const* sockets = getenv(socket_takeover);
+    dbgln("eccoci {}", sockets);
     if (!sockets) {
         s_overtaken_sockets_parsed = true;
         return;
     }
 
-    for (auto& socket : StringView { sockets, strlen(sockets) }.split_view(' ')) {
+    for (auto& socket : StringView { sockets, strlen(sockets) }.split_view(';')) {
```

Comment:
Take `auto const` instead of `auto&`, small types are better copied than referenced

Reply:
Thanks! Improved!